core/templates/services/svg-sanitizer.service.ts

```diff
@@ -183,9 +183,14 @@ export class SvgSanitizerService {
         node.remove();
       } else {
         for (let i = 0; i < node.attributes.length; i++) {
-          let nodeAttrName: string = node.attributes[i].name.toLowerCase();
+          const nodeAttrName: string = node.attributes[i].name;
+          const nodeAttrCombination = (
+            nodeTagName.toLocaleLowerCase() + ':' + nodeAttrName.toLowerCase());
           if (
-            attrsToBeRemoved.indexOf(nodeTagName + ':' + nodeAttrName) !== -1) {
+            attrsToBeRemoved.findIndex(
+              attribute => attribute.toLowerCase() === nodeAttrCombination
```

Comment:
Can you explain why we need to normalize everything to lowercase in the first place? Are SVG tag/attr names not case-sensitive?